crates/testing/tests/tests_cx_hardening.rs

```diff
@@ -0,0 +1,3 @@
+mod tests_cx_hardening {
+    automod::dir!("tests/tests_cx_hardening");
```

Comment:
I think `tests_cx_hardening` would make a task namespace into a first-class feature of the codebase, muddying the circumstances for when we should change them. Instead, let's just add these under the existing da task tests and remove this special file.

Reply:
+1 to just throw this into one of the test groups